scroll/readme.md

```diff
@@ -0,0 +1,153 @@
+# Scroll API
+
+## General
+The Scroll API allows Logz.io users to run queries on the data in their account via REST (i.e. HTTP). The query language used is Elasticsearch Search API DSL, with certain limitations. We highly recommend you read the restrictions listed below (In the Request Body section) before beginning to use the API
+
+## License 
+Using the API requires a special license from Logz.io, and an API token which can be generated here: https://app.logz.io/#/dashboard/settings/shared-tokens 
+
+The number of queries executed is controlled and limited by Logz.io.
+
+The number of results are limited to 1,000.
+
+## Scroll Command
+
+### URL
+URL: https://api.logz.io/v1/scroll (US Accounts)
+     https://api-eu.logz.io/v1/scroll (EU Accounts)
+* Make sure to use https and not http, otherwise you will get a '403 Forbidden' response..
+
+### HTTP Headers
+* "X-USER-TOKEN" -  The value should be the API token you generated (as explained in the License section above)
+* "Content-Type" - "application/json"  (required)
+
+### Request Body 
+Elasticsearch Search API Body as documented  in [Elaticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.x/search.html), with the following restrictions:
+* We support the following top-level elements in the Elasticsearch Search API:  `query`, `from`, `size`, `sort`, `_source`, `post_filter`, `scroll`, `scroll_id`.
+* When using the `query_string` element, you are not allowed to set its field named `allow_leading_wildcard` to true
+* When using the `wildcard` element, you are not allowed to have its value start with `*` or `?`
+* When using the `scroll_id` element, you are not allowed to use `query` element.
+* Your query will only run on data you sent today and yesterday. You can still specify a filter on the timestamp field to allow smaller time frame.
```

Comment:
I don't think it is updated (in the previous search API as well). Need to verify, but I think that the query will run on today and yesterday by default, but you can still contact support and ask to extend it if needed